### DISCUSSION TITLE
v-data-table sort only asc and desc

### DISCUSSION THREAD

**Discussion by MalteKiefer** [vuetifyjs/vuetify#14357] [Q&A] [answered]
Hello,

just want to know if it is possible to modify the sort function so, that it is only possible to sort for ASC or DESC.
Currently, you have 3 possibilities, DESC, ASC and no sort.

Thank you.

**Comment by KaelWD** [comment, marked as answer]
https://vuetifyjs.com/en/api/v-data-table/#props-must-sort
